src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/AesGcm.Unix.cs

```diff
@@ -111,7 +111,7 @@ private void ImportKey(ReadOnlySpan<byte> key)
 
             if (plaintextBytesWritten != plaintext.Length)
             {
-                Debug.Fail($"GCM decrypt wrote {plaintextBytesWritten} of {plaintext.Length} bytes.");
+                 // Debug.Fail($"GCM decrypt wrote {plaintextBytesWritten} of {plaintext.Length} bytes.");
```

Comment:
I'll work on trying to isolate and fix this issue, but in the meantime this is the easiest way to keep this `Debug.Fail` method from taking down the test process.

Reply:
Unlike ECB and CBC, GCM produces one byte of output for one byte of input.  Goodness isn't happening if this assert fails (and tests seemingly shouldn't be passing, unless they got the data from elsewhere having depended on this assert)

Reply:
If this is needed, please don't merge without creating an issue to put it back.

Reply:
Oh the tests aren’t passing. But this commenter out assert allows them to run and fail and have XUnit record the issue.

Reply:
They fail from the exception that immediately follows this assert.

Reply:
Tracking issue for re-enabling the assert is here: https://github.com/dotnet/runtime/issues/48471